### PR TITLE
Checkstyle bumped to version 6.14.1

### PR DESCRIPTION
**Pull request by rafalnowak**
Checkstyle needs to be updated to version 6.14.1 because there were new rules and in current version the most recent google style did not work.

### CODE REVIEW

File: src/main/java/pl/touk/sputnik/processor/checkstyle/CheckstyleProcessor.java [hunk 1]
```diff
@@ -45,7 +45,11 @@ public String getName() {
     private void innerProcess(@NotNull Review review, @NotNull AuditListener auditListener) {
         List<File> files = review.getFiles(new JavaFilter(), new IOFileTransformer());
         Checker checker = createChecker(auditListener);
-        checker.process(files);
+        try {
+            checker.process(files);
+        } catch(CheckstyleException e) {
```
Comment (sputnikci) [inline src/main/java/pl/touk/sputnik/processor/checkstyle/CheckstyleProcessor.java]: [Checkstyle] ERROR: 'catch' is not followed by whitespace.

Comment (SpOOnman) [inline src/main/java/pl/touk/sputnik/processor/checkstyle/CheckstyleProcessor.java]: +1

**Comment by SpOOnman** [comment]
Thank you @rafalnowak. Fix the issue and it's ready to merge.

File: src/main/java/pl/touk/sputnik/processor/checkstyle/CheckstyleProcessor.java (hunk 1 above)
Comment (pjagielski) [inline src/main/java/pl/touk/sputnik/processor/checkstyle/CheckstyleProcessor.java]: Sputnik works!

**Comment by SpOOnman** [comment]
Thanks!

**Pull request merged**